server/README.md

```diff
@@ -57,35 +57,29 @@ enum Tee {
 
 #### GetPolicy
 
-Get the specific TEE's OPA `Policy(.rego)`. The message:
+Get the OPA `Policy(.rego)`. The message:
 ```PROTO
 message GetPolicyRequest {
-    // Supported TEE types
-    Tee tee = 1;
     // Optional: Designate the user id. It should be kept as "None" currently.
     optional string user = 2;
```

Comment:
This fields remains `= 2` to avoid confusing client that already used `1` for the `tee` field?
(same question regarding all messages here)